chainer/training/extensions/__init__.py

```diff
@@ -1,7 +1,7 @@
 # import classes and functions
 from chainer.training.extensions._snapshot import snapshot  # NOQA
 from chainer.training.extensions._snapshot import snapshot_object  # NOQA
-from chainer.training.extensions.computational_graph import dump_graph  # NOQA
+from chainer.training.extensions.computational_graph import DumpGraph  # NOQA
```

Comment:
Could you leave the old name for backward compatibility? It's ok to define it as an alias to `DumpGraph`.

Reply:
Thank you.
I added the areas for backward compatibility in last commit.